form-config/default-1.0-draft.js

```diff
@@ -434,16 +435,21 @@ module.exports = {
                     }
                   }
                 },
+                //-----------------------------
+                // Data manager
+                //------------------------------
                 {
                   class: 'ContributorField',
                   showHeader: true,
                   definition: {
                     name: 'contributor_data_manager',
-                    required: true,
+                    required: false,
```

Comment:
Why are we changing data manager to no longer be required?

Reply:
Hi Simon, the required attribute is just a normal CSS class (hence concerned with frontend only). It doesn't interfere with the actual logic. Besides the typescript version is due to some packages in my local. It concerns with what angular version I used for my dev environment. Hence, it should not be an issue as such.